youtube_dl/extractor/canvas.py

```diff
@@ -53,6 +53,15 @@ def _real_extract(self, url):
                     'url': format_url,
                 })
         self._sort_formats(formats)
+        
+        subtitles = {}
+        for target in data['subtitleUrls']:
```

Comment:
Even when it's _always_ there?
for example: https://mediazone.vrt.be/api/v1/canvas/assets/mz-ast-a30df0cf-590c-4c4f-b1fe-7c28fc3b7b8c